src/System.Private.CoreLib/shared/System/InsufficientMemoryException.cs

```diff
@@ -26,8 +25,10 @@ namespace System
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed class InsufficientMemoryException : OutOfMemoryException
     {
+        // There may be a problem here interacting with the ResourceManager in out of memory conditions,
+        // CoreCLR can fall back to literal strings.
         public InsufficientMemoryException()
-            : base(GetMessageFromNativeResources(ExceptionMessageKind.OutOfMemory))
```

Comment:
If GetMessageFromNativeResources method is not used after this change, delete it.

Reply:
OK. it wouldn't be used after this.